driver/ppm_fillers.c

```diff
@@ -360,13 +360,13 @@ int f_sys_open_x(struct event_filler_arguments *args)
 	uint32_t dev = 0;
 	uint64_t ino = 0;
 	int res;
-	int64_t retval;
+	s32 retval;
 
 	/*
 	 * fd
 	 */
-	retval = (int64_t)syscall_get_return_value(current, args->regs);
-	res = val_to_ring(args, retval, 0, false, 0);
+	retval = (s32)syscall_get_return_value(current, args->regs);
+	res = val_to_ring(args, (s64)retval, 0, false, 0);
```

Comment:
we don't need this on `retval`, it is really a `long` so we don't need this cast

Reply:
My bad!